ql/src/java/org/apache/hadoop/hive/ql/Context.java

```diff
@@ -117,6 +117,7 @@ public class Context {
   protected String optimizedSql;
   protected String calcitePlan;
   protected String cmd = "";
+  public static String compactPartition = "hive.iceberg.compaction.partition";
```

Comment:
that shouldn't belong here, maybe AlterTableCompactOperation?

Reply:
Done, moved to AlterTableCompactOperation.